modules/navigation/2d/nav_mesh_generator_2d.cpp

```diff
@@ -660,6 +662,58 @@ void NavMeshGenerator2D::generator_parse_tilemap_node(const Ref<NavigationPolygo
 	}
 }
 
+void NavMeshGenerator2D::generator_parse_navigationobstacle_node(const Ref<NavigationPolygon> &p_navigation_mesh, Ref<NavigationMeshSourceGeometryData2D> p_source_geometry_data, Node *p_node) {
```

Comment:
It adds 2x projected obstacles. 1x for the radius and 1x for the vertices.

Reply:
oh, so that's expected... actually, now that I think of it maybe it makes sense not to rule out such a situation as it causes no harm - fair enough then

Reply:
Yes because the radius is always centered on the obstacle position while the vertices can be anywhere and both can be sized independently. If you have just moved the obstacle you also might have both active for a short time.